sources/zendesk/README.md

```diff
@@ -1,294 +1,66 @@
-    """
-    Loads all possible tables for Zendesk Support, Chat, Talk
-    """
-    # FULL PIPELINE RUN
-    pipeline = dlt.pipeline(pipeline_name="dlt_zendesk_pipeline", destination="bigquery", full_refresh=True, dataset_name="sample_zendesk_data3")
-
-    # zendesk support source function
-    data_support = zendesk_support(load_all=True)
-    # zendesk chat source function
-    data_chat = zendesk_chat()
-    # zendesk talk source function
-    data_talk = zendesk_talk()
-    # run pipeline with all 3 sources
-    info = pipeline.run(data=[data_support, data_chat, data_talk])
-    return info
-```
-
-## Run the pipeline
-
-1. Install requirements for the pipeline by running the following command:
-
-`pip install -r requirements.txt`
-
-2. Run the pipeline with the following command:
-
-`python3 zendesk_pipeline.py`
-
-3. To make sure everything is loaded as expected, use the command:
-
-`dlt pipeline zendesk_pipeline show`
-
-
-# Customizations
-
-The Zendesk pipeline has some default customizations that make it more useful:
-
-1. **Pivoting ticket fields:** By default, the source pivots the custom fields in tickets when loading the data, which allows the custom fields to be used as columns after loading. This behavior is due to the fact that the boolean parameter `pivot_ticket_fields` in the source method `zendesk_support()` is set to `True` by default. To change this, set `pivot_ticket_fields=False` when calling the source method from inside the data loading function.
-```python
-data_support = zendesk_support(pivot_ticket_fields=False)
-```
-
-  Alternatively, this can be explicitly done by using the function `load_support_with_pivoting` in the script `zendesk_pipeline.py`.
-  ```python
-  def load_support_with_pivoting():
-    """
-    Loads Zendesk Support data with pivoting. Simply done by setting the pivot_ticket_fields to true - default option. Loads only the base tables.
-    """
-    pipeline = dlt.pipeline(pipeline_name="zendesk_support_pivoting", destination='bigquery', full_refresh=False)
-    data = zendesk_support(load_all=False, pivot_ticket_fields=True)
-    info = pipeline.run(data=data)
-    return info
-  ```
-  Simply include this function in the `__main__` block when running the pipeline.
-  ```python
-  if __name__ == "__main__":
-    load_support_with_pivoting()
-  ```
-
-2. **Custom field rename**: The source loads the custom fields with their label names. If the label changes between loads, the initial label continues to be used (is persisted in state). To reset the labels, do a full refresh. This can be achieved by passing `full_refresh=True` when calling `dlt.pipeline`:
-```python
-pipeline = dlt.pipeline(pipeline_name="dlt_zendesk_pipeline", destination='bigquery', full_refresh=True, dataset_name="sample_zendesk_data3")
-```
+Here, we chose BigQuery as the destination. Alternatively, you can also choose Redshift, duckdb, or any of the other [destinations](https://dlthub.com/docs/dlt-ecosystem/destinations/).
```

Comment:
we should write all destination in lowercase or all of them with a capital letter. uniformity.
in previous documents you wrote in lowercase)

Reply:
done, lower case is better.